### ISSUE TITLE
missing diacritics

### ISSUE THREAD

**Issue by paulens12** [Julow/Unexpected-Keyboard#128]
this keyboard makes it impossible to type ąęėįų, also note the difference between ę and ȩ (the keyboard only has the latter)

**Comment by Julow** [comment]
I would be happy to add these. Do you need these for the Latvian language ?

**Comment by paulens12** [comment]
> I would be happy to add these. Do you need these for the Latvian language ?

Lithuanian :) I think Latvian needs ķ and similar consonants which also don't seem to be available.

**Comment by Julow** [comment]
I asked because the diacritics are associated to languages, to avoid showing all of them to everyone. Latvian's `ķ` should be `¸`+`k`. 

The new diacritics will be available in the next release :) fec3f10

**Comment by paulens12** [comment]
Thanks! I don't speak Latvian but I don't think they use the same diacritics. The only ones I've seen were ķīš

**Comment by Julow** [comment]
I think these can be typed on the keyboard. If you want to try the Lithuanian support before it is released, just in case, you can find a debug build here: https://github.com/Julow/Unexpected-Keyboard/actions/runs/2213702707

**Comment by paulens12** [comment]
that's OK, I just wanted to point out that you added these new diacritics to qwerty_lv layout, where they're not actually needed, because the Latvian language doesn't use them.

**Comment by Julow** [comment]
I've done that because the `accent_...` keys are supposed to be on every layouts. The new diacritics won't be shown to users of `qwerty_lv` that don't also have Lithuanian enabled in their settings.

I didn't add them to every other layouts because I plan on improving the way the diacritics are shown on the keyboard soon. But I added them to the qwerty_lv layout until then, because the countries are close to each other.

**Comment by Julow** [comment]
TL;DR: It's an implementation detail, Latvian users won't see Lithuanian diacritics.
